src/System.IO.Pipelines/src/System.IO.Pipelines.csproj

```diff
@@ -0,0 +1,55 @@
+  <PropertyGroup>
+    <ProjectGuid>{1032D5F6-5AE7-4002-A0E4-FEBEADFEA977}</ProjectGuid>
+    <DocumentationFile>$(OutputPath)$(MSBuildProjectName).xml</DocumentationFile>
+    <DefineConstants Condition="'$(TargetGroup)'=='netcoreapp2.1'">$(DefineConstants);netcoreapp21</DefineConstants>
+    <DefineConstants Condition="'$(TargetGroup)'=='netstandard1.1'">$(DefineConstants);netstandard11</DefineConstants>
+    <DefineConstants Condition="'$(TargetGroup)'=='netstandard'">$(DefineConstants);netstandard</DefineConstants>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="Properties\InternalsVisibleTo.cs" />
+    <Compile Include="System\IO\Pipelines\BufferSegment.cs" />
+    <Compile Include="System\IO\Pipelines\FlushResult.cs" />
+    <Compile Include="System\IO\Pipelines\InlineScheduler.cs" />
+    <Compile Include="System\IO\Pipelines\IPipeAwaiter.cs" />
+    <Compile Include="System\IO\Pipelines\IPipeConnection.cs" />
+    <Compile Include="System\IO\Pipelines\Pipe.DefaultPipeReader.cs" />
+    <Compile Include="System\IO\Pipelines\Pipe.DefaultPipeWriter.cs" />
+    <Compile Include="System\IO\Pipelines\Pipe.cs" />
+    <Compile Include="System\IO\Pipelines\PipeAwaitable.cs" />
+    <Compile Include="System\IO\Pipelines\PipeAwaiter.cs" />
+    <Compile Include="System\IO\Pipelines\PipeCompletion.cs" />
+    <Compile Include="System\IO\Pipelines\PipeCompletionCallback.cs" />
+    <Compile Include="System\IO\Pipelines\PipeCompletionCallbacks.cs" />
+    <Compile Include="System\IO\Pipelines\PipeOptions.cs" />
+    <Compile Include="System\IO\Pipelines\PipeReader.cs" />
+    <Compile Include="System\IO\Pipelines\PipeReaderState.cs" />
+    <Compile Include="System\IO\Pipelines\PipeScheduler.cs" />
+    <Compile Include="System\IO\Pipelines\PipeWriter.cs" />
+    <Compile Include="System\IO\Pipelines\ReadResult.cs" />
+    <Compile Include="System\IO\Pipelines\ResultFlags.cs" />
+    <Compile Include="System\IO\Pipelines\ThrowHelper.cs" />
+  </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)'=='netcoreapp2.1'">
+    <Compile Include="System\IO\Pipelines\ThreadPoolScheduler.NetCoreApp21.cs" />
+  </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)'=='netstandard1.1'">
+    <Compile Include="System\IO\Pipelines\ThreadPoolScheduler.NetStandard.cs" />
+  </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)'=='netstandard'">
+    <Compile Include="System\IO\Pipelines\ThreadPoolScheduler.NetStandard20.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <Reference Include="System.Buffers" />
+    <Reference Include="System.Collections" />
+    <Reference Include="System.Diagnostics.Debug" />
+    <Reference Include="System.Memory" />
+    <Reference Include="System.Resources.ResourceManager" />
+    <Reference Include="System.Runtime" />
+    <Reference Include="System.Runtime.Extensions" />
+    <Reference Include="System.Threading.Tasks" />
```

Comment:
You are also missing a reference to `System.Threading.ThreadPool` in order to get the WaitCallback